include/experimental/__p0009_bits/extents.hpp

```diff
@@ -142,7 +142,7 @@ class extents
   template <size_t... Idxs>
   MDSPAN_FORCE_INLINE_FUNCTION
   static constexpr
-  index_type _static_extent_impl(size_t n, std::integer_sequence<size_t, Idxs...>) noexcept {
+  std::size_t _static_extent_impl(size_t n, std::integer_sequence<size_t, Idxs...>) noexcept {
     return _MDSPAN_FOLD_PLUS_RIGHT(((Idxs == n) ? Extents : 0), /* + ... + */ 0);
```

Comment:
```suggestion
    return _MDSPAN_FOLD_PLUS_RIGHT(((Idxs == n) ? Extents : 0), /* + ... + */ size_t(0));
```
?

Reply:
Discussed; this should not be necessary.

Reply:
ah I see what you mean, no you don't need that cast their in the end, you do need the extra argument though